inc/cloner/class-cloner.php

```diff
@@ -1343,6 +1343,10 @@ protected function cloneMetadata() {
 				}
 			} elseif ( $key === 'pb_title' ) {
 				update_post_meta( $metadata_post_id, $key, $this->targetBookTitle );
+			} elseif ( $key === 'pb_institutions' ) {
+				foreach ( $value as $institution ) {
+					add_post_meta( $metadata_post_id, $key, $institution );
+				}
```

Comment:
Instead of checking for the key, we could maybe do something like this:
```php
 elseif ( is_array( $key ) ) {
    foreach ( $value as $institution ) {
        add_post_meta( $metadata_post_id, $key, $institution );
    }
}
```